spring/src/main/java/uk/ac/bbsrc/tgac/miso/spring/ajax/PoolControllerHelperService.java

```diff
@@ -808,15 +658,15 @@ public JSONObject createElementSelectDataTable(HttpSession session, JSONObject j
         for (LibraryDilution libraryDilution : requestManager.listAllLibraryDilutionsByPlatform((PlatformType.get(platform)))) {
           JSONArray pout = new JSONArray();
           pout.add(libraryDilution.getName());
-          pout.add(libraryDilution.getConcentration());
+          pout.add(String.format("%.1f", libraryDilution.getConcentration()));
```

Comment:
I prefer consistent numbers of decimal places, but prod seems to have up to five decimal places and I refuse to apply that standard to everything else. I'll change it so it's not limited to any decimal places.